tests/integration/test_render.py

```diff
@@ -194,3 +202,21 @@ def test_background_image(self):
         image_path = '/resource/drag-and-drop-v2/public/img/triangle.png'
         self.assertTrue(bg_image.get_attribute("src").endswith(image_path))
         self.assertEqual(bg_image.get_attribute("alt"), 'This describes the target image')
+
+    def test_borders_inactive(self):
+        self.load_scenario()
+        zones = self._get_zones()
+        for index, zone in enumerate(zones):
+            elt = 'zone-{}'.format(index + 1)
+            # Firefox does not provide a single "border-style", so we check "border-top-style"
+            self.assertEqual(self._get_border_style(elt, 'border-top-style'), 'none')
+            self.assertEqual(self._get_border_style(elt, 'border-top-width'), '0px')
+
+    def test_borders_active(self):
+        self.load_scenario("", "", True)
+        zones = self._get_zones()
+        for index, zone in enumerate(zones):
+            elt = 'zone-{}'.format(index + 1)
+            self.assertEqual(self._get_border_style(elt, 'border-top-style'), 'dotted')
+            self.assertEqual(self._get_border_style(elt, 'border-top-width'), '1px')
+            self.assertEqual(self._get_border_style(elt, 'border-top-color'), 'rgb(86, 86, 86)')  # == #565656
```

Comment:
@agaylard For readability, can you add another color (such as `DARK_GREY`) to the `Colors` class above and check against that here (instead of specifying the RGB and hex values here)?